material/elements/vaadin-date-picker.html

```diff
@@ -0,0 +1,218 @@
+<link rel="import" href="vaadin-button.html">
+<link rel="import" href="vaadin-text-field.html">
+
+
+<dom-module id="material-date-picker" theme-for="vaadin-date-picker">
+  <template>
+    <style>
+       :host {
+        width: 250px;
+        max-width: 100%;
+      }
+
+      [part="text-field"] {
+        font-size: var(--material-font-size-m);
+        max-width: 100%;
+        color: var(--material-body-text-color);
+        margin-bottom: 0;
+      }
+
+      [part="clear-button"],
+      [part="toggle-button"] {
+        cursor: pointer;
+        color: var(--material-secondary-text-color);
+        font-size: var(--material-font-size-l);
+      }
+
+      [part="clear-button"]:hover,
+      [part="toggle-button"]:hover {
+        color: var(--material-body-text-color);
+      }
+    </style>
+  </template>
+</dom-module>
+
+<dom-module id="material-date-picker-month-calendar" theme-for="vaadin-month-calendar">
+  <template>
+    <style include="vaadin-month-calendar-default-theme">
+      [part~="date"] {
+        position: relative;
+        height: 32px;
+        font-weight: normal;
+
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      [part~="date"][focused]:not([disabled])::after {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        margin: auto;
+        z-index: -1;
+        content: '';
+        height: 32px;
+        width: 32px;
+        border-radius: 100%;
+        opacity: 0.5;
+        background: var(--material-primary-color);
+      }
+
+      :host([focused]) [part~="date"][focused]:not([disabled])::after {
+        opacity: 1;
+      }
+
+      [part~="date"][today] {
+        color: var(--material-primary-text-color);
+      }
+
+      [part~="date"][selected] {
+        font-weight: bold;
+      }
+
+      :host([focused]) [part~="date"][focused]:not([disabled]) {
+        background: transparent;
+      }
+
+      [part~="date"][focused]:not([disabled]) {
+        color: var(--material-primary-contrast-color);
+        background: transparent;
+        box-sizing: border-box;
+      }
+
+      [part~="date"][selected] {
+        font-weight: 600;
+      }
+
+      [part~="weekdays"] {
```

Comment:
Isn’t this selector unnecessary? Move the font size property (and use the correct value, not `small`) to the following selector (weekday) and remove the color, as it’s overridden by the next selector.